src/dart-2.0.md

```diff
@@ -0,0 +1,138 @@
+description: How Dart 2.0 is different than 1.x, and how you can prepare.
+---
+
+This page has information about changes that are coming in Dart 2.0.
+
+<aside class="alert alert-info" markdown="1">
+**Flutter:** You don't need to do anything to prepare Flutter code for Dart 2.0.
+If changes become necessary, watch this page for updates.
+</aside>
+
+Migration tips for:
+
+* [Web](#migration-tips-web)
+* [Servers and command-line scripts](#migration-tips-servers-and-command-line-scripts)
+
+What's different in Dart 2.0:
+
+* [Obsolete features](#obsolete features)
+* [Strong mode and static typing](#strong-mode-and-static-typing)
+
+## Migration tips: web
+
+If you develop for the web, you can start migrating to Dart 2.0 now:
+
+Convert your code to be strong mode compliant
+: For more information,
+  see [Strong mode and static typing](#strong-mode-and-static-typing) below.
+
+Optimize the structure of your libraries
+: Place library code that won't be _directly_ imported by your `web` code,
+  or by other packages, under `lib/src`. For more information,
+  see the documentation for [creating library
+  packages.](/guides/libraries/create-library-packages)
+  You can find examples of this directory structure in the [Stagehand
+  templates.](https://github.com/google/stagehand/tree/master/templates)
+
+{% comment %}
+Save this section for when dartdevc is ready to use...
+Adding strong mode support to the Dart analyzer (introduced in Dart 1.15),
+made it possible to create a new development compiler,
+[dartdevc]({{site.webdev}}/tools/dartdevc) (also known as _DDC_).
+You can use dartdevc on strong mode-compliant
+code to run and debug your Dart web apps in Chrome or other browsers.
+With the introduction of dartdevc,
+[Dartium](http://news.dartlang.org/2017/06/a-stronger-dart-for-everyone.html)
+will eventually be phased out.
+{% endcomment %}
+
+## Migration tips: servers and command-line scripts
+
+If you develop servers or command-line scripts,
+you can start migrating to Dart 2.0 now:
+
+* Convert your code to be strong mode compliant. For more information,
+  see [Strong mode and static typing](#strong-mode-and-static-typing) below.
+
+## Obsolete features
+
+The following features are redundant and will not be supported by Dart 2.0:
+
+Checked mode
+: Strong mode replaces checked mode. To learn how they are different, see
+  [What is the difference between strong mode and checked
+  mode?](/guides/language/sound-faq#how-is-it-different-than-checked-mode)
+
+Dartium
+: The [dartdevc]({{site.webdev}}/tools/dartdevc) compiler
+  performs fast compiles to JavaScript,
+  thanks to strong mode. Instead of Dartium, you’ll use Chrome
+  or other standard browsers for testing. For information on
+  Dartium's removal, see [A stronger Dart for
+  everyone.](http://news.dartlang.org/2017/06/a-stronger-dart-for-everyone.html)
+
+## Strong mode and static typing
+
+<aside class="alert alert-info" markdown="1">
+**Note about Flutter:**
```

Comment:
Use the same **bold text** you have in the first note. I think starting with "Flutter" is probably a good idea, for scannability. I suggested **Flutter developer?** but **Flutter note:** would work, too.